stout_a02b.js

```diff
@@ -0,0 +1,24 @@
+/*Kyle Stout
+  stout_a02b
+  INFO 2124WW
+  Thoendel
+  12-15-2019
+*/
+
+let firstLastName = 'Kyle Stout';
+let myAge = 28;
```

Comment:
this variable shouldn't support block scoping - that indicates we should declare with the var keyword.
-2